### PR TITLE
fix(statics): update hbar explorer url

### PR DESCRIPTION
**Pull request by adarsh312**
Ticket: WIN-2452

### CODE REVIEW

File: modules/statics/src/networks.ts [hunk 1]
```diff
@@ -535,7 +535,7 @@ class EosTestnet extends Testnet implements AccountNetwork {
 class Hedera extends Mainnet implements AccountNetwork {
   name = 'Hedera';
   family = CoinFamily.HBAR;
-  explorerUrl = 'https://explorer.kabuto.sh/mainnet/transaction/';
+  explorerUrl = 'https://app.dragonglass.me/hedera/search?q=';
```
Comment (mukeshsp) [inline modules/statics/src/networks.ts:538]: I think, this should be `https://app.dragonglass.me/transactions/`

Comment (aarvay) [reply modules/statics/src/networks.ts:538]: +1

Comment (ajays97) [inline modules/statics/src/networks.ts:538]: We can also use https://hashscan.io, just saying. But anything is fine. Please fix as per @mukeshsp 's comment.

Comment (adarsh312) [reply modules/statics/src/networks.ts:538]: I checked a sample transaction, the dragonglass headers are different from transaction id 
attaching a sample to check it 

https://app.dragonglass.me/transactions/008350931710747032564252074

Comment (ajays97) [reply modules/statics/src/networks.ts:538]: [HashScan](https://hashscan.io) works fine with txn hash, check below link for example. It redirects to the correct txn page with ID

[https://hashscan.io/mainnet/transaction/dea1c57c78199f9e576b7de0487f3c3d32dbb65218887662cbc3c3750e90e047ae9d5272402fca40f73a574a90c618ba](https://hashscan.io/mainnet/transaction/dea1c57c78199f9e576b7de0487f3c3d32dbb65218887662cbc3c3750e90e047ae9d5272402fca40f73a574a90c618ba)

Comment (grsind19) [reply modules/statics/src/networks.ts:538]: @ajays97  can you test for our testnet txn hashes, the hashscan says it is not found.

Comment (adarsh312) [reply modules/statics/src/networks.ts]: @ajays97  updated it to hashScan. please review

**Review by ajays97 — APPROVED** [review]
Explorer URL changes look good for HBAR mainnet.

**Pull request merged**